dali/operators/reader/tfrecord_reader_op.cc

```diff
@@ -53,14 +54,16 @@ DALI_SCHEMA(_TFRecordReader)
 // Schema for the actual TFRecordReader op exposed
 // in Python. It is here for proper docstring generation
 DALI_SCHEMA(TFRecordReader)
-  .DocStr(R"code(Read sample data from a TensorFlow TFRecord file.)code")
+  .DocStr(R"code(Reads the sample data from a TensorFlow TFRecord file.)code")
   .AddArg("features",
-      R"code(Dictionary of names and configuration of features existing in TFRecord file.
-Typically obtained using helper functions `dali.tfrecord.FixedLenFeature`
-and `dali.tfrecord.VarLenFeature`, they are equivalent to TensorFlow's `tf.FixedLenFeature` and
-`tf.VarLenFeature` respectively. For more flexibility `dali.tfrecord.VarLenFeature`
-supports `partial_shape` parameter. If provided, data will be reshaped to match its value.
-First dimension will be inferred from the data size.)code",
+      R"code(Dictionary of names and configuration of features that exist in the TFRecord file.
```

Comment:
This doesn't explain anything...
```suggestion
      R"code(A dictionary that maps names of the TFRecord features to extract to the feature type.
```
Or pehaps something even more elaborate.

Reply:
Done